scaffolding-ruby/lib/scaffolding.sh

```diff
@@ -393,7 +393,16 @@ scaffolding_create_dir_symlinks() {
 }
 
 scaffolding_create_files_symlinks() {
-  return 0
+  local entry source_file target
+
+  for entry in "${scaffolding_symlinked_files[@]}"; do
+    source_file="$scaffolding_app_prefix/$entry"
+    target="$pkg_svc_path/$entry"
```

Comment:
@fnichol at this point I think I would be leaning towards switching to doing the `scaffolding_symlinked_configs`.  At least as a starting point.